dashboard/config/foorm/library/surveys/pd/ayw_satisfaction_panel.0.json

```diff
@@ -6,6 +6,8 @@
     {
      "type": "panel",
      "name": "workshop_satisfaction_panel",
+          "title": "How much do you agree or disagree with the following statements about the workshop overall?",
```

Comment:
the title should be on the matrix instead of the panel

Reply:
facepalm.

Reply:
missed adding this change?